.github/workflows/build.yml

```diff
@@ -101,7 +101,7 @@ jobs:
           OPERATOR_ID: '0.0.1022'
           HEDERA_NETWORK: 'localhost'
         run: |
-          ./gradlew -POPERATOR_ID=$OPERATOR_ID -POPERATOR_KEY=$OPERATOR_KEY -PHEDERA_NETWORK=$HEDERA_NETWORK :aggregation:testCodeCoverageReport --scan
+          ./gradlew -POPERATOR_ID=$OPERATOR_ID -POPERATOR_KEY=$OPERATOR_KEY -PHEDERA_NETWORK=$HEDERA_NETWORK :aggregation:testCodeCoverageReport --scan --rerun-tasks
```

Comment:
The `--rerun-tasks` should not be added here. It will lead to recompiling the code and not reuse it from the step before. It will also not use anything from the build cache making the remote cache more or less useless. What is the reason for adding it?

Reply:
Alright

Reply:
I just wanted to make sure the tests are actually running